edx_proctoring/views.py

```diff
@@ -168,6 +170,68 @@ def handle_exception(self, exc):
         return resp
 
 
+class ProctoredExamAttemptsMFEView(ProctoredAPIView):
+    """
+    Endpoint for getting timed or proctored exam and its attempt data.
+    /edx_proctoring/v1/proctored_exam/exam_attempts/course_id/{}/content_id/(?P<content_id>{})
+
+    Supports:
+        HTTP GET: Returns an existing exam (by course_id and content id) with latest related attempt.
+        and active attempt for any exam in progress (for the timer feature)
+        {
+            'exam': {
+                ...
+                attempt: { ... }
+            },
+            'active_attempt': { ... },
+        }
+    """
+
+    def get(self, request, course_id, content_id):
+        """
+        HTTP GET handler. Returns exam with attempt and active attempt
+        """
+        active_attempt_data = {}
+        attempt_data = {}
+        active_exam = {}
+
+        active_exams = get_active_exams_for_user(request.user.id)
+        if active_exams:
+            # Even if there is more than one exam, we want the first one.
+            # Normally this should not be an issue as there will be list of one item.
+            active_exam_info = active_exams[0]
+            active_exam = active_exam_info['exam']
+            active_attempt = active_exam_info['attempt']
+            active_attempt_data = get_exam_attempt_data(
+                active_exam.get('id'),
+                active_attempt.get('id'),
+                is_learning_mfe=True
```

Comment:
can we have this pass as a query parameter so this function is in frontend specific?